src/masternode-payments.cpp

```diff
@@ -333,15 +330,21 @@ void CMasternodePayments::ProcessMessage(CNode* pfrom, std::string& strCommand,
 
     } else if (strCommand == NetMsgType::MASTERNODEPAYMENTVOTE) { // Masternode Payments Vote for the Winner
 
+
         CMasternodePaymentVote vote;
         vRecv >> vote;
 
-        if(pfrom->nVersion < GetMinMasternodePaymentsProto()) return;
-
         uint256 nHash = vote.GetHash();
 
         pfrom->setAskFor.erase(nHash);
 
+        // TODO: clear setAskFor for MSG_MASTERNODE_PAYMENT_BLOCK too
```

Comment:
not needed because if node is misbehaving by sending wrong peer version let setAskFor be cleared.